src/js/Gameboard.js

```diff
@@ -0,0 +1,105 @@
+import {Ship, allShips} from "./Ship";
+
+const Gameboard = () => {
+  let board = new Array(100).fill(0);
+  const COLUMN = 10;
+  const SLOT = {
+    EMPTY: 0,
+    MISS: 1,
+    HIT: 6
+  };
+
+  const randomPlacement = () => {
+
+    const direction = () => {
+      let hv = ['horizontal', 'vertical'];
+      let direction = Math.floor(Math.random() * hv.length);
+      return hv[direction];
+    };
+
+    const getShipsName = () => {
+      let shipName = [];
+      for (let ship in allShips) {
+        shipName.push(ship);
+      }
+      return shipName;
+    };
```

Comment:
My idea for this: What about using `Object.keys(allShips)` ?

Reply:
It's another way to do the same thing, at that very moment It wasn't in my mind